htetl/tasks/loadpages.py

```diff
@@ -0,0 +1,11 @@
+import luigi
+from  htetl import util
+
+class RawPageData(util.QueryPostgres):
+    """ load eaxmple """
+    sql = 'select Id, Content from page limit 100'
```

Comment:
Maybe we should only dump `Id`, because dumping HTML into a CSV file and reading it back correctly is messy business?

@dlrobertson thoughts?

Reply:
Yes it is messy, but we do need a way to snag the html and make it available for future tasks. To be honest, not sure what the best way forward is here.

Reply:
What if you saved each page as a separate file with the name of the pageId? Another option might be to store it in a local database like sqlite3 and work the etl off of that. It's a bit redundant, but may be quicker if the database is on another server. There would be quite the overhead for both options, but it's less messy to read into memory

Reply:
> Another option might be to store it in a local database like sqlite3 and work the etl off of that. It's a bit redundant, but may be quicker if the database is on another server. There would be quite the overhead for both options, but it's less messy to read into memory

Yeah, I like both of those better than a csv. Either one works, but I think I like the sqlite3 db better. @bmenn thoughts?